RELEASE_NOTES.rst

```diff
@@ -104,9 +108,6 @@ Breaking changes:
 * ``quamash`` has been replaced with ``qasync``.
 * Protocols are updated to use device endian.
 * Analyzer dump format includes a byte for device endianness.
-* To support variable numbers of Urukul cards in the future, the
-  ``artiq.coredevice.suservo.SUServo`` constructor now accepts two device name lists,
-  ``cpld_devices`` and ``dds_devices``, rather than four individual arguments.
```

Comment:
This should not be removed from the ARTIQ-6 release notes.

Reply:
I think this was a mistake - these changes were not yet part of c22482787e (ARTIQ-6 tag). Sorry for missing this in the last PR.